card/mod/04_settings/set/abstract/permission.rb

```diff
@@ -0,0 +1,99 @@
+def standardize_items
+  # noop to override default behavior, which wouldn't let '_left' through and would therefore break
+end
+
+def testoo
+  'bestoo'
+end
+
+format :html do
+
+  view :pointer_core do |args| #, :view=>:core, :mod=>Type::Pointer::HtmlFormat
+    %{<div class="pointer-list">#{ render_pointer_items args }</div>}
```

Comment:
the PR obscures this change, but this is the only real change since this code was in comment.rb.  It breaks if you use the view aliasing syntax, since abstract sets are loaded before anything else.  We may need to come up with a way to postpone aliasing until after everything is loaded so mods aren't so sensitive to load order.  ...or a way to express dependencies ?